packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -1,23 +1,27 @@
+import { fireEvent, screen } from '@testing-library/react';
 import React from 'react';
 import SimpleSchema from 'simpl-schema';
-import { AutoForm, connectField } from 'uniforms';
+import { AutoForm, connectField, context } from 'uniforms';
 import { SimpleSchema2Bridge } from 'uniforms-bridge-simple-schema-2';
+import { AutoFields } from 'uniforms-unstyled';
 
-import mount from './_mount';
+import { render } from '../__suites__';
 
-describe('AutoForm', () => {
-  const onChangeModel = jest.fn();
-  const validator = jest.fn();
-  const onChange = jest.fn();
-  const onSubmit = jest.fn();
-  const model = { a: '1' };
+describe('<AutoForm />', () => {
   const schema = new SimpleSchema2Bridge(
     new SimpleSchema({
       a: { type: String, defaultValue: '' },
       b: { type: String, defaultValue: '' },
       c: { type: String, defaultValue: '' },
     }),
   );
+  const model = { a: '1' };
```

Comment:
There's no point in touching this code - please revert it.

Reply:
fixed